### PR TITLE
Task/riui 726

### PR DESCRIPTION
**Pull request by mike-suggitt**

### CODE REVIEW

File: src/app/routing/routing.module.ts [hunk 1]
```diff
@@ -129,11 +121,11 @@ const routes: Routes = [
         HearingConfirmationComponent
     ],
     providers: [
-        CaseService,
+        // CaseService,
```
Comment (cksanders) [inline src/app/routing/routing.module.ts]: please delete commented out code.

File: src/app/routing/routing.module.ts [hunk 2]
```diff
@@ -129,11 +121,11 @@ const routes: Routes = [
...
         CaseResolve,
         DecisionResolve,
         RedirectionService,
-        QuestionService,
+        // QuestionService,
```
Comment (cksanders) [inline src/app/routing/routing.module.ts]: please delete commented out code.

File: src/app/routing/pages/view-case/view-case.component.ts [hunk 3]
```diff
@@ -11,11 +11,47 @@ import { ActivatedRoute } from '@angular/router';
...
+                    label: section.name,
+                    id: section.id
+                };
+            });
+        }
+        this.targetSection = (this.case) ? this.case.sections.find(section => section.id === this.sectionId) : null;
+        if (!this.targetSection) {
+            if (this.links[0]) {
+                this.router.navigate([ this.links[0].id], {relativeTo: this.route})
+                    .then(
```
[the remark this answers is not in the document]
Comment (mike-suggitt) [reply src/app/routing/pages/view-case/view-case.component.ts]: Good suggestion. will update

File: src/app/routing/resolve/decision.resolve.ts [hunk 4]
```diff
@@ -11,9 +11,7 @@ export class DecisionResolve implements Resolve<any> {
     }
 
     resolve(route: ActivatedRouteSnapshot) {
-        console.log('resolve');
-        return this.decisionService.fetch(route.paramMap.get('case_id'))
-            .pipe(catchError(error => {
+        return this.decisionService.fetch(route.paramMap.get('case_id')).pipe(catchError(error => {
```
Comment (cksanders) [inline src/app/routing/resolve/decision.resolve.ts]: try to put chained methods on new line -> this allows devs to put blocks when debugging.

Comment (mike-suggitt) [reply src/app/routing/resolve/decision.resolve.ts]: Yes I agree however unfortunately this also has a significant impact on code coverage :(

**Pull request merged**